### ISSUE TITLE
Allow custom path for gpg

### ISSUE THREAD

**Issue by smira** [aptly-dev/aptly#699]
As `gpg2` is becoming default now, allow custom path/name for gpg1 binaries.

**Comment by smira** [comment]
From https://github.com/smira/aptly/issues/689#issuecomment-360898037

**Comment by eighthave** [comment]
I'm getting this with only GnuPG 2.2 installed:

https://salsa.debian.org/android-tools-team/android-platform-external-libselinux/-/jobs/27568
```
Setting up gpg (2.2.8-3) ...
Setting up gpg-agent (2.2.8-3) ...
Setting up gpg-wks-server (2.2.8-3) ...
Setting up gpg-wks-client (2.2.8-3) ...
Setting up gnupg (2.2.8-3) ...
Setting up aptly (1.3.0-2) ...
Processing triggers for libc-bin (2.27-3) ...
aptly set to automatically installed.
gnupg set to automatically installed.
Config file not found, creating default config at /root/.aptly.conf


Local repo [autobuilt] successfully added.
You can run 'aptly repo add autobuilt ...' to add packages to repository.
panic: Couldn't find a suitable gpg executable. Make sure gnupg1 is available as either gpg or gpg1 in $PATH [recovered]
	panic: Couldn't find a suitable gpg executable. Make sure gnupg1 is available as either gpg or gpg1 in $PATH

goroutine 1 [running]:
github.com/aptly-dev/aptly/cmd.Run.func1(0xc420303f58)
	/build/aptly-yRHGWT/aptly-1.3.0/obj-x86_64-linux-gnu/src/github.com/aptly-dev/aptly/cmd/run.go:17 +0x11a
panic(0xb94c20, 0xc420304310)
	/usr/lib/go-1.10/src/runtime/panic.go:502 +0x229
github.com/aptly-dev/aptly/pgp.NewGpgVerifier(0xc4202da0c0)
	/build/aptly-yRHGWT/aptly-1.3.0/obj-x86_64-linux-gnu/src/github.com/aptly-dev/aptly/pgp/gnupg.go:178 +0x12c
github.com/aptly-dev/aptly/context.(*AptlyContext).GetVerifier(0xc4202da0c0, 0x0, 0x0)
	/build/aptly-yRHGWT/aptly-1.3.0/obj-x86_64-linux-gnu/src/github.com/aptly-dev/aptly/context/context.go:415 +0xce
github.com/aptly-dev/aptly/cmd.aptlyRepoAdd(0xc4201d7680, 0xc4202d3850, 0x7, 0x7, 0xd5b000, 0xc420276c80)
	/build/aptly-yRHGWT/aptly-1.3.0/obj-x86_64-linux-gnu/src/github.com/aptly-dev/aptly/cmd/repo_add.go:23 +0x73
github.com/aptly-dev/aptly/vendor/github.com/smira/commander.(*Command).Dispatch(0xc4201d7680, 0xc4202d3850, 0x7, 0x7, 0xc420303e28, 0x56587a)
	/build/aptly-yRHGWT/aptly-1.3.0/obj-x86_64-linux-gnu/src/github.com/aptly-dev/aptly/vendor/github.com/smira/commander/commands.go:305 +0x288
github.com/aptly-dev/aptly/vendor/github.com/smira/commander.(*Command).Dispatch(0xc42028c5a0, 0xc4202d3840, 0x8, 0x8, 0xcc5678, 0xc42014e000)
	/build/aptly-yRHGWT/aptly-1.3.0/obj-x86_64-linux-gnu/src/github.com/aptly-dev/aptly/vendor/github.com/smira/commander/commands.go:283 +0x135
github.com/aptly-dev/aptly/vendor/github.com/smira/commander.(*Command).Dispatch(0xc42030a7e0, 0xc4202d3830, 0x9, 0x9, 0xc420311340, 0xc4202d3830)
	/build/aptly-yRHGWT/aptly-1.3.0/obj-x86_64-linux-gnu/src/github.com/aptly-dev/aptly/vendor/github.com/smira/commander/commands.go:283 +0x135
github.com/aptly-dev/aptly/cmd.Run(0xc42030a7e0, 0xc42001e0b0, 0x9, 0x9, 0x1, 0x0)
	/build/aptly-yRHGWT/aptly-1.3.0/obj-x86_64-linux-gnu/src/github.com/aptly-dev/aptly/cmd/run.go:41 +0x129
main.main()
	/build/aptly-yRHGWT/aptly-1.3.0/obj-x86_64-linux-gnu/src/github.com/aptly-dev/aptly/main.go:24 +0x151
ERROR: Job failed: exit code 1
```

**Comment by smira** [comment]
@eighthave nightly has workaround https://github.com/aptly-dev/aptly/issues/657#issuecomment-398210315

**Comment by smira** [comment]
Should be fixed with #779
